frontend/src/hooks/users.ts

```
@@ -15,4 +15,12 @@ export const fetchUserLogin = async (user: LoginRequest) => {
     } catch (error: any) {
         throw error;
     }
+};
+
+export const fetchUserLogout = async () => {
+    try {
+        await axios.post('http://localhost/api/logout');
+    } catch (error: any) {
+        throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)